xds/internal/xdsclient/transport/loadreport.go

```diff
@@ -160,15 +160,17 @@ func (t *Transport) sendLoads(ctx context.Context, stream lrsStream, clusterName
 			return
 		}
 		if err := t.sendLoadStatsRequest(stream, t.lrsStore.Stats(clusterNames)); err != nil {
-			t.logger.Warningf("Failed to write to LRS stream: %v", err)
+			t.logger.Warningf("Writing to LRS stream: %v", err)
 			return
 		}
 	}
 }
 
 func (t *Transport) sendFirstLoadStatsRequest(stream lrsStream, node *v3corepb.Node) error {
 	req := &v3lrspb.LoadStatsRequest{Node: node}
-	t.logger.Debugf("Sending initial LoadStatsRequest: %s", pretty.ToJSON(req))
+	if t.logger.V(perRPCVerbosityLevel) {
```

Comment:
`Debugf` is `V(2) && Info()`.  If increasing the verbosity level of these logs, we should also switch to `Infof` instead.

Also, consider a wrapper instead to avoid repeating this pattern a lot.  Maybe `t.logger` can override `Debugf` to be level 9?  Or add a `Trace` level or something.  Or maybe we should just change `grpclog.Debugf` to use `9` everywhere?

Reply:
I changed all calls to `Debugf` inside this verbosity check to `Infof`. I didn't want to change the default verbosity level of `Debugf` to `9` since I still want to retain the ability to get some debug logs, but not all (especially the ones corresponding to every request/response). Thanks.